src/gui/entry/EntryModel.cpp

```diff
@@ -20,7 +20,9 @@
 #include <QFont>
 #include <QMimeData>
 #include <QPalette>
+#include <zconf.h>
```

Comment:
What is this for?

Reply:
Good Question. Must have slipped in from me experimenting, i removed it